src/storage/src/controller.rs

```diff
@@ -194,9 +203,19 @@ pub trait StorageController: Debug + Send {
     /// Create the sinks described by the `ExportDescription`.
     async fn create_exports(
         &mut self,
-        exports: Vec<(GlobalId, ExportDescription<Self::Timestamp>)>,
+        exports: Vec<(CreateExportToken, ExportDescription<Self::Timestamp>)>,
     ) -> Result<(), StorageError>;
 
+    /// Notify the storage controller to prepare for an export to be created
+    async fn pending_export(
```

Comment:
bikeshedding: maybe `prepare_export` (to stay with the two-phase-commit-y smell of this), and then `cancel_prepared_export()`?